functions/go/fmt/README.md

```diff
@@ -0,0 +1,73 @@
+# fmt
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Format resources using openapi schema.
```

Comment:
> using openapi schema

I feel this makes it confusing as short description.

Reply:
Updated.